src/Microsoft.NET.Workloads/Microsoft.NET.Workloads.csproj

```diff
@@ -16,6 +16,8 @@
     <!-- See: https://devblogs.microsoft.com/nuget/add-a-readme-to-your-nuget-package/ -->
     <PackageReadmeFile>README.md</PackageReadmeFile>
     <ReadmeFile>$(IntermediateOutputPath)README.md</ReadmeFile>
+    <PackageProjectUrl>https://learn.microsoft.com/dotnet/core/tools/dotnet-workload-sets</PackageProjectUrl>
```

Comment:
I would've thought this would be a link to this repo? I saw some packages that did that and always found it useful, but maybe that's just me. Maybe there should be two URLs.

Reply:
Especially since the docs link is in the description 🙂

Reply:
Where's the source URL defined? I don't see it in this file

Reply:
It's automatically injected by source link. Check any of the packages already on NuGet and you'll see it there already - it's one of the nicer things we got into dotnet 8 :)

Reply:
Nice! Definitely a win.